lib/potentialbuild.rb

```diff
@@ -406,7 +410,38 @@ def next_build
     @acting_as_baseline = false
   end
 
-  def parse_call_grind(build_dir, file)
+  def parse_file_sizes(file)
```

Comment:
Alright, so this function reads a single file that is a list of file sizes.  As of now I'm not sure how that file is created, but that's fine.